src/shared-components/Card/ListItems/LisItemsNotification.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  List,
+  ListItem,
+  ListItemSecondaryAction,
+  ListItemText,
+  Typography,
+} from '@mui/material';
+import { colors } from 'src/styles/colors';
+import { formatTime } from 'src/utils/timeUtils';
+
+export const ListItemsNotification = ({
+  title,
+  description,
+  time,
+}: {
+  title: string;
+  description: string;
+  time: string | Date;
+}) => {
+  const formattedTime = formatTime(time);
+  return (
+    <List>
```

Comment:
Remove list element, only keep listItem. Don't forget to clean imports.

Reply:
Fixed